api/opencl_platform_layer.asciidoc

```diff
@@ -1154,6 +1154,21 @@ include::{generated}/api/version-notes/CL_DEVICE_SUB_GROUP_INDEPENDENT_FORWARD_P
 |====
 
 4::
+    OpenCL adopters must report a valid vendor ID for their implementation.
+    If there is no valid PCI vendor ID defined for the physical device,
+    implementations must obtain a Khronos vendor ID. To reserve a Khronos vendor
+    ID, propose a merge request against cl.xml in the master branch of the
+    OpenCL-Docs project. The merge must define a new enumerant by adding an
```

Comment:
Do you want to say something about the awkward "reserve in vk.xml and mirror to cl.xml" strategy here?

Reply:
Good point, part of this awkwardness is that it won't be appropriate to reserve the new cl.xml vendor ID in vk.xml before mirroring if a vendor doesn't implement Vulkan.

Added these sentences in a new commit:

> To keep Khronos vendor IDs synchronized across standards, the proposed ID value should not collide with the ID of another vendor already reserved in vk.xml, from the master branch of the Vulkan-Docs project. If an adopter has already reserved a Khronos vendor ID in vk.xml, then that value should be duplicated in cl.xml.

Reply:
That is a situation which seems avoidable, thanks for the possible solutions. 

In favour of splitting the ranges since it mitigates the cross repository interaction, and requiring reservation in vk.xml first feels like it violates [this Vulkan spec wording](https://www.khronos.org/registry/vulkan/specs/1.2/styleguide.html#extensions-vendor-id)

> Please do not try to reserve vendor IDs unless you are making a good faith effort to develop a Vulkan implementation and require one for that purpose.

Will update this PR to reflect that.

Reply:
Actually, after some discussion internally we're going to revisit mandating the id be reserved in vk.xml and come back to this PR

Reply:
I think we could alter the Vulkan spec wording you quote, to allow for a global Khronos vendor ID. The intent wasn't to prevent useful cross-API functionality.

Reply:
We've accepted an MR internally that allows reserving vendor IDs for other Khronos APIs in vk.xml, and that will go out in the next spec update, next week. Since that's locked now, this can move forward - we plan to publish a spec update next Tuesday (03/17).

Reply:
great thanks, pushed new language to reflect that